features/Functional-parts.md

```diff
@@ -25,4 +25,5 @@
 | SYS  |                                  | @SYS        |                                                  |
 | IFC  | IFC                              | @IFC        | Validating correct IFC Schema                    |
 | PSE  | Property Sets                    | @PSE        |                                                  |
-| APA  |                                  | @PSE        |                                                  |
\ No newline at end of file
+| APA  |                                  | @PSE        |                                                  |
```

Comment:
What's APA?

Reply:
No idea, it was already in there. Maybe something related to Alignment? 

If you don't know, maybe we can remove it?